File: cgi-bin/hello.pl
#!/usr/bin/perl
use strict;
use warnings;

my $dateString = gmtime();
my $content = "<html><head><title>Demo Perl Script</title></head><body><h2>GMT date/time $dateString</h2></body></html>\r\n";

print "HTTP/1.0 200 OK\r\n";
print "Content-type: text/html\r\n";
print "Content-length: " . length($content) . "\r\n";
print "\r\n";
print $content;

File: cgi-bin/hello.rb
#! /bin/ruby

content = "<html><head><title>Demo Ruby Script</title></head><body><h1>What a ruby script it is huh...</h1></body></html>"

puts "HTTP/1.1 200 OK"
puts "Content-type: text/html"
puts "Content-length: #{content.length}"
puts ""
puts content